src/core/distributions/phylogenetics/substitution/AbstractPhyloCTMCSiteHomogeneous.h

```diff
@@ -934,9 +937,9 @@ double RevBayesCore::AbstractPhyloCTMCSiteHomogeneous<charType>::computeLnProbab
             size_t right_index = right.getIndex();
             fillLikelihoodVector( right, right_index );
 
+            dirty_nodes[root_index] = false;
```

Comment:
Previously we weren't marking the partial likelihoods at the root as clean when we computed them.